gitlab/cli.py

```diff
@@ -53,8 +53,9 @@ def register_custom_action(
     cls_names: Union[str, Tuple[str, ...]],
     mandatory: Tuple[str, ...] = tuple(),
     optional: Tuple[str, ...] = tuple(),
+    custom_action: Optional[str] = None,
 ) -> Callable[[__F], __F]:
-    def wrap(f: __F) -> __F:
+    def wrap(f: __F, custom_action: Optional[str] = custom_action) -> __F:
```

Comment:
I don't think this function definition needs to be updated.

`__F` already declares it a callable that takes any number of arguments.

Reply:
Oops, thanks for this :)